internal/rest/election_period.go

```diff
@@ -0,0 +1,79 @@
+package rest
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/Coflnet/sky-mayor/internal/model"
+	"github.com/Coflnet/sky-mayor/internal/mongo"
+	"github.com/gin-gonic/gin"
+	"github.com/rs/zerolog/log"
+)
+
+func InitElectionPeriodREST() {
```

Comment:
you can move that to the rest.Init Func

Reply:
I thought this would be cleaner if I add more rest endpoints later?